From b5acde022fd7d1682a8337e91e64ab9ed4e107fd Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Thu, 9 Jul 2020 15:05:08 +0700
Subject: [PATCH 333/441] dt-bindings: power: rcar-sysc: Document r8a774a3 sysc

Document bindings for the RZ/G2M v3.0 (a.k.a. R8A774A3)
system controller.
Also update name for RZ/G2M v1.3 (a.k.a. R8A774A1) system controller.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 Documentation/devicetree/bindings/power/renesas,rcar-sysc.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/power/renesas,rcar-sysc.txt b/Documentation/devicetree/bindings/power/renesas,rcar-sysc.txt
index 9194e69..30da5c0 100644
--- a/Documentation/devicetree/bindings/power/renesas,rcar-sysc.txt
+++ b/Documentation/devicetree/bindings/power/renesas,rcar-sysc.txt
@@ -10,7 +10,8 @@ Required properties:
       - "renesas,r8a7743-sysc" (RZ/G1M)
       - "renesas,r8a7745-sysc" (RZ/G1E)
       - "renesas,r8a77470-sysc" (RZ/G1C)
-      - "renesas,r8a774a1-sysc" (RZ/G2M)
+      - "renesas,r8a774a1-sysc" (RZ/G2M v1.3)
+      - "renesas,r8a774a3-sysc" (RZ/G2M v3.0)
       - "renesas,r8a774b1-sysc" (RZ/G2N)
       - "renesas,r8a774c0-sysc" (RZ/G2E)
       - "renesas,r8a774e1-sysc" (RZ/G2H)
-- 
2.7.4

